Backend/dev/migrations/rider-app/1107-create-ride-location-table-and-add-columns-to-ride-table.sql

```diff
@@ -0,0 +1,26 @@
+-- before the release
+CREATE TABLE atlas_app.trip_location as SELECT * FROM atlas_app.booking_location;
+ALTER TABLE atlas_app.trip_location ADD CONSTRAINT trip_location_pkey PRIMARY KEY (id);
+
+-- before the release
+ALTER TABLE atlas_app.ride ADD COLUMN from_location_id character(36) NOT NULL;
+ALTER TABLE atlas_app.ride ADD COLUMN to_location_id character(36);
```

Comment:
Why is `to_location_id` nullable here but not in driver app?

Reply:
It is because the rental slab does not have any toLocation so this column has to be NOT NULL.